flag_timestamp.go

```diff
@@ -0,0 +1,130 @@
+package cli
+
+import (
+	"flag"
+	"fmt"
+	"time"
+)
+
+
+// timestamp wrap to satisfy golang's flag interface.
+type timestampWrap struct {
```

Comment:
```suggestion
type Timestamp struct {
```

All the flags expose the wrapped type to allow setting a zero value default.